digraph "unitGraph" {
    "r5 := @parameter0: java.lang.String[]"
    "r0 = newarray (java.security.ProtectionDomain)[1]"
    "$r1 = new java.security.AccessControlContext"
    "specialinvoke $r1.<java.security.AccessControlContext: void <init>(java.security.ProtectionDomain[])>(r0)"
    "virtualinvoke $r1.<java.security.AccessControlContext: void checkPermission(java.security.Permission)>(null)"
    "$r2 = new java.lang.Exception"
    "specialinvoke $r2.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"
    "throw $r2"
    "$r3 := @caughtexception"
    "r4 = $r3"
    "return"
    "r5 := @parameter0: java.lang.String[]"->"r0 = newarray (java.security.ProtectionDomain)[1]";
    "r0 = newarray (java.security.ProtectionDomain)[1]"->"$r1 = new java.security.AccessControlContext";
    "r0 = newarray (java.security.ProtectionDomain)[1]"->"$r3 := @caughtexception";
    "$r1 = new java.security.AccessControlContext"->"specialinvoke $r1.<java.security.AccessControlContext: void <init>(java.security.ProtectionDomain[])>(r0)";
    "$r1 = new java.security.AccessControlContext"->"$r3 := @caughtexception";
    "specialinvoke $r1.<java.security.AccessControlContext: void <init>(java.security.ProtectionDomain[])>(r0)"->"virtualinvoke $r1.<java.security.AccessControlContext: void checkPermission(java.security.Permission)>(null)";
    "specialinvoke $r1.<java.security.AccessControlContext: void <init>(java.security.ProtectionDomain[])>(r0)"->"$r3 := @caughtexception";
    "virtualinvoke $r1.<java.security.AccessControlContext: void checkPermission(java.security.Permission)>(null)"->"$r2 = new java.lang.Exception";
    "virtualinvoke $r1.<java.security.AccessControlContext: void checkPermission(java.security.Permission)>(null)"->"$r3 := @caughtexception";
    "$r2 = new java.lang.Exception"->"specialinvoke $r2.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")";
    "$r2 = new java.lang.Exception"->"$r3 := @caughtexception";
    "specialinvoke $r2.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"throw $r2";
    "specialinvoke $r2.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"$r3 := @caughtexception";
    "throw $r2"->"$r3 := @caughtexception";
    "$r3 := @caughtexception"->"r4 = $r3";
    "r4 = $r3"->"return";
}
